src/dashboard/components/SubmissionEntry.tsx

```diff
@@ -37,10 +37,7 @@ const SubmissionEntry: React.FC<Props> = ({
                 className={`submission-entry__link submission-entry__link--${status}`}
                 to={submission.lastStepVisited}
                 onClick={(e: React.MouseEvent): void => {
-                    if (
-                        status !== 'continue_submission' ||
-                        (!submission.articleType || submission.articleType === '')
-                    ) {
+                    if (status !== 'continue_submission' || !submission.articleType || submission.articleType === '') {
```

Comment:
Just want to double check, the logic here has changed slightly from 'This or one of these'  to 'this or this or this'. Was that intentional?

Reply:
I think the end logic here remains the same. Regardless I've tested manually and it all works as intended so I'm happy for it to go in as is.